src/features/onboarding/Welcome.js

```diff
@@ -20,11 +20,12 @@ const Welcome = () => {
   const profile = useSelector(selectProfile)
 
   const welcomeCompleted = Boolean(parseInt(profile.welcomeCompleted))
-  const newsletterCompleted = profile.newsletter !== undefined
+  const addToHomeScreen = Boolean(parseInt(profile.welcomeCompleted))
```

Comment:
To keep with the style rename to `addToHomeScreenCompleted` and then you need to read a new value from profile. I would do `profile.addToHomeScreenCompleted`. You now read `profile.welcomeCompleted` so you will never get the home screen prompt.